File: app/src/main/java/databinding/example/com/java/util/MyBindingMethods.java
package databinding.example.com.java.util;

import android.databinding.BindingMethod;
import android.databinding.BindingMethods;
import android.widget.ImageView;

@BindingMethods(value = {
        @BindingMethod(
                type = ImageView.class, attribute = "app:srcCompat", method = "setImageResource"),
})
public class MyBindingMethods {
}

File: app/src/main/java/databinding/example/com/basic/util/MyBindingMethods.kt
package databinding.example.com.basic.util

import android.databinding.BindingMethod
import android.databinding.BindingMethods
import android.widget.ImageView

@BindingMethods(
        BindingMethod(type = ImageView::class,
                attribute = "app:srcCompat",
                method = "setImageResource"))
class MyBindingMethods
